hooks/useFuturesData.ts

```diff
@@ -293,6 +295,7 @@ const useFuturesData = () => {
 
 	useEffect(() => {
 		resetTradeState();
+		dispatch(fetchStakingData());
```

Comment:
I think the best place for this is to be added to `usePollMarketFuturesData` in `state/futures/hooks` you can probably add a line like:

`useFetchAction(fetchStakingData, { changeKeys: [networkId, wallet] });`

Reply:
Sure. I will make the changes.